packages/labs/eslint-plugin/src/test/rules/rule-tester.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {RuleTester} from '@typescript-eslint/rule-tester';
+import {describe as suite, test} from 'node:test';
```

Comment:
If using the node test runner, we should bump the node engine in package.json to `16.17.0` which is when it was added.
https://github.com/lit/lit/blob/546d75d9e26f4aefa3bf91d114a70a3c3fb05da2/package.json#L6

Reply:
Noted, though there's a difference between the Node version required for development and version required to run the library code. I'm not sure what version of Node eslint, typescript-eslint, or our code here require actually.

Reply:
Ah, excellent point about dev requirement being different. Dependency packages _should_ have their own engines specified, maybe? In any case, no action needed with regards to the original comment.